sde_collections/models/collection.py

```diff
@@ -242,12 +246,12 @@ def create_scraper_config(self, overwrite: bool = False):
         if overwrite is True, it will overwrite the existing file
         """
 
-        scraper_template = open("config_generation/xmls/webcrawler_initial_crawl.xml").read()
+        scraper_template = open("config_generation/xmls/initial_config_template.xml").read()
```

Comment:
change the names to `scraper_template` and `indexer_template`. The way you do this in git is copy over the files first without the rename and commit it so that git knows. Then you do the rename as another commit.